modules/vpc-endpoints/terraform.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
```

Comment:
fixes:
> │ Warning: Reference to undefined provider
│ 
│   on main.tf line 256, in module "vpc_endpoints_use1":
│  256:     aws = aws.use1